### DISCUSSION TITLE
Tables not getting created by `ecto.migrate`

### DISCUSSION THREAD

**Discussion by restlessronin** [vinniefranco/exandra#52] [Q&A] [answered]
I can run `ecto.migrate` successfully now (only after `ecto.create` is called). But when I look at the db with DataGrip, the only table I see in the keyspace is `schema_migrations`. This table is empty, probably indicating that no migrations have been run.

There are 3 migrations in priv/repo/migrations, but those tables have not been created.

The code is at https://github.com/cyberchitta/M.ai

**Comment by whatyouhide** [comment, marked as answer]
The issue there is that your migration files are `.ex` files when they must be `.exs` files. Generate migrations with `mix ecto.gen.migration` to avoid this issue. 👍

**Reply by restlessronin** [reply]
Duh 😵‍💫 . Thank you @whatyouhide .
